datasets/us/mi/med_exclusions/crawler.py

```diff
@@ -91,9 +91,9 @@ def crawl_item(row: Dict[str, str], context: Context):
 def crawl_excel_url(context: Context):
     doc = context.fetch_html(context.data_url)
     doc.make_links_absolute(context.data_url)
-    return doc.xpath("//*[text()='List of Sanctioned Providers (XLSX)']/../..")[0].get(
-        "href"
-    )
```

Comment:
I think we can write this more readably and robustly with with `/ancestor::a` and and a variable